loops.js

```diff
@@ -51,3 +67,14 @@
 // >> 9
 // ==========================================
 
+for(let i = 0; i<10; i++) {
+
+            if (i<3)
+                console.log(i);
+            if ((i>=3) && (i<6))
```

Comment:
Net als bij regel 72, vergeet niet de accolades te gebruiken na de ronde haakjes in de if-statement.
Ook 2 x ronde haakjes bij regel 74 zijn niet nodig, de ronde haakjes die je voor de if() gebruikt zijn voldoende:

if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }